src/main/java/com/logdyn/ui/console/commands/HelpCommand.java

```diff
@@ -19,6 +21,31 @@ public HelpCommand(final Collection<Command> commands) {
 
     @Override
     public void execute(final String... args) {
+        if (args.length == 0) {
+            this.commands.forEach(this::printHelp);
+        } else if (args[0].equalsIgnoreCase("web")) {
+            openBrowser();
+        } else {
+            final Optional<Command> command = this.commands.stream()
+                    .filter(c -> c.getNames().contains(args[0].toLowerCase()))
+                    .findAny();
+            if (command.isPresent())
+            {
+                command.ifPresent(this::printHelp);
+            }
+            else
+            {
+                DefaultCommand.execute(args[0]);
+            }
+        }
+    }
+
+    private void printHelp(final Command command){
+        System.out.printf("[%s] %s - %s%n", String.join(" | ", command.getNames()), command.getDescription(), command.getHelp());
```

Comment:
Could maybe use a comment to show an example output

Reply:
done